src/controller.js

```diff
@@ -435,7 +435,8 @@ export default ['$scope', '$element', function ($scope, $element) {
             form = currentMes;
           }
           var mes = '';
-          var split = form.split(aggr[i]);
+          var aggrFunc = form.match(new RegExp(aggr[i], 'i'));
```

Comment:
is this necessary? Can't you just use aggr[i] as before? I might be wrong, but it seems to be the same thing.

Reply:
This has been used todisplay the same case as user has entered in the Master Item like sum or sUM etc. if we use agg[0] then its the standard Sentence case i.e. Sum for all the cases.